### PR TITLE
EdgesGeometry: Update docs and minor clean up

### PR DESCRIPTION
**Pull request by Mugen87**
* Added parameter `thresholdAngle` to the docs
* Use of `Float32BufferAttribute`
* Added a comment that explains the application of `thresholdAngle`

### CODE REVIEW

**Comment by Mugen87** [comment]
@WestLangley Can you please review the comment. I'm not sure about the correct wording...

File: docs/api/geometries/EdgesGeometry.html [hunk 1]
```diff
@@ -14,40 +14,23 @@
...
 		</code>
 
 		<h2>Constructor</h2>
 
-		<h3>[name]( [page:Geometry geometry] )</h3>
+		<h3>[name]( [page:Geometry geometry], [page:Integer thresholdAngle] )</h3>
 		<div>
-		geometry — any geometry object.
+		geometry — Any geometry object.<br />
+		thresholdAngle — A valid edge is only detected if the angle between two consecutive faces is greater than this value, default = 1.
```
Comment (WestLangley) [inline docs/api/geometries/EdgesGeometry.html]: thresholdAngle — An edge is only rendered if the angle (in degrees) between the face normals of the two adjoining faces exceeds this value. default = 1 degree.

File: src/geometries/EdgesGeometry.js [hunk 2]
```diff
@@ -76,6 +76,9 @@ function EdgesGeometry( geometry, thresholdAngle ) {
 
 		var h = hash[ key ];
 
+		// A valid edge is only detected if the angle between two consecutive faces is greater than the given threshold angle.
+		// The implementation detects this via the dot product of two consecutive faces normals.
```
Comment (WestLangley) [inline src/geometries/EdgesGeometry.js]: Rewrite as noted above.

**Comment by Mugen87** [comment]
Yep, that's better 👍

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**